src/cloud/components/DocTagsListItem.tsx

```diff
@@ -91,15 +91,15 @@ const DocTagsListItem = ({
 const Container = styled.div`
   display: inline-flex;
   flex-wrap: wrap;
-  padding: 0.25em 0.5em;
+  padding: 0 0.5em;
```

Comment:
if possible, avoid em and other such values in the future
pixels are more reliable and won't possibly alter displays depending on the screen size

Reply:
I will not use em in the future